recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -22,6 +22,10 @@ DEPENDS:append:mx8-nxp-bsp = " patchelf-native"
 
 DEPENDS:append:libc-musl = " gcompat"
 
+DEPENDS:remove:imx-nxp-bsp = "mesa"
+PACKAGES:remove:imx-nxp-bsp = "libgl-imx libgl-imx-dev"
+PROVIDES:remove:imx-nxp-bsp = "virtual/libgl"
```

Comment:
These removals were done explicitly at the last update: https://github.com/nxp-upstream/meta-freescale/commit/364fa4a6416758ae8c47f3165336caae6951fe36

Reply:
@valijec Is it possible to send another fix for this?

Reply:
> @valijec Is it possible to send another fix for this?

I opened a PR for the fixup #2006

> This change doesn't do anything since these removals were done explicitly at the last update: [nxp-upstream@364fa4a](https://github.com/nxp-upstream/meta-freescale/commit/364fa4a6416758ae8c47f3165336caae6951fe36)

Thank you, you're right it does not take effect :)